apps/in_hand_scanner/include/pcl/apps/in_hand_scanner/impl/common_types.hpp

```diff
@@ -80,23 +80,31 @@ namespace pcl
         this->directions = 0;
       }
 
-      inline PointIHS (const PointIHS& other)
+      inline PointIHS& operator=(const PointIHS& other)
```

Comment:
Could we just `default` it or skip mentioning it at all?

Reply:
I think setting the copy assignment function to `default` is a quite better choice, although refactoring those functions works equally. I fixed it in https://github.com/PointCloudLibrary/pcl/pull/4695/commits/00e2f43d8b8494ea2cf954761fb8c95c94fb7b7c. :smiley: